src/app/home/mentors/mentors.component.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import {
+  TranslateModule,
+  TranslateLoader,
+  TranslateFakeLoader
+} from '@ngx-translate/core';
+import {of} from 'rxjs';
+
+import { LanguageService } from '../../core/services/language.service';
+import { MentorService } from './mentor.service';
+import { MentorsComponent } from './mentors.component';
```

Comment:
Just as a comment. When ordering your imports, also take into consideration the path. Nearer path first.